src/precompile.jl

```diff
@@ -0,0 +1,30 @@
+using SnoopPrecompile: SnoopPrecompile
+
+const __TEST_NOTEBOOK_ID = uuid1()
+
+module __Foo end
+
+SnoopPrecompile.@precompile_all_calls begin
+    let
+        channel = Channel{Any}(10)
+        Pluto.PlutoRunner.setup_plutologger(
+            __TEST_NOTEBOOK_ID,
+            channel,
+        )
+    end
+    expr = Expr(:toplevel, :(1 + 1))
+    Pluto.PlutoRunner.run_expression(__Foo, expr, __TEST_NOTEBOOK_ID, uuid1(), nothing);
```

Comment:
It would be nice if this precompilation could benefit notebook processes too (it is only called for simple markdown cells on the Pluto process), maybe with #1881 ?

Reply:
I just tried to add the following to `src/precompile.jl`:
```julia
    session = Pluto.ServerSession()
    session.options.evaluation.workspace_use_distributed = false
    basic = joinpath(pkgdir(Pluto), "sample", "Basic.jl")
    nb = load_notebook(basic)
    Pluto.WorkspaceManager.make_workspace((session, nb))
```
This grinds my PC to a half. GUI becomes unresponsive even though the compilations occurs on only one thread. I suspect that the calls to `Distributed` or some of the other async calls are the problem.